src/views/Home/index.view.tsx

```diff
@@ -25,13 +27,36 @@ const description = [
   },
 ];
 
+const navBarButtons = {
+  hacker: {
+    text: "Apply to hack!",
+    link: "/",
+    label: "hacker sign-up button",
+  },
+  sponsor_mentor: {
+    text: "Be a sponsor / mentor!",
+    link:
+      "https://docs.google.com/forms/d/e/1FAIpQLScVmW-gIcKGrp7fBVeH5i8G1VQ83IwmKrkIo_J2PeRj3PRYTw/viewform",
+    label: "sponsor sign-up button",
+  },
+};
+
```

Comment:
Here is the link to apply to hack: https://bit.ly/cruzhacks-2021-hacker

Reply:
@ilopezro - Ahh, yeah! Sorry about the Freudian slip on the judge/mentor/sponsor linking. Regarding the dictionary versus array implementation, I like how dictionaries name our props, but you do make a good point in how `map()` can reduce verbosity.

I'm implemented a hybrid approach, wherein we iterate through the values of a dictionary, so let me know how it looks!

Reply:
I really like what you did here -- this looks awesome! My only concern is that `Navbar` component, but I think I'm ok racking up a bit of technical debt for now since we want these buttons out!